test/OpenSTValue.js

```diff
@@ -506,4 +508,250 @@ contract('OpenSTValue', function(accounts) {
 			})
 		})
 	})
+
+
+	// Revert Staking before ProcessStaking
+	describe('Process RevertStaking before ProcessStaking', async () => {
+
+		context('Revert Stake', async () => {
+
+			var staker = accounts[0];			
+			var owner  = accounts[1];
+			var amountST = 126;
```

Comment:
We should do full scan of unit tests of consistent use of ST vs STWei (UT vs UTWei, BT vs BTWei)